src/meta/src/hummock/compaction_schedule_policy.rs

```diff
@@ -68,7 +70,7 @@ pub trait CompactionSchedulePolicy: Send + Sync {
 // This strategy is retained just for reference, it is not used.
 pub struct RoundRobinPolicy {
     /// The context ids of compactors.
-    compactors: Vec<HummockContextId>,
+    compactors: Vec<(HummockContextId, /* is not frozen */ bool)>,
```

Comment:
How about use a dedicated Vec for freeze_compactors ?

Reply:
> use a dedicated Vec for freeze_compactors

thanks. fixed.